appinfo/routes.php

```diff
@@ -281,6 +281,15 @@
 				'token' => '^[a-z0-9]{4,30}$',
 			],
 		],
+		[
+			'name' => 'Room#setConversationState',
+			'url' => '/api/{apiVersion}/room/{token}/conversation-state',
```

Comment:
how about reusing the "readOnly" attribute and the "/read-only" endpoint but with the new flag value ?

Reply:
Yeah sounds like a good idea